.github/workflows/bpf-checks.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Run checkpatch.pl
-        uses: docker://cilium/cilium-checkpatch:e6a6843cef50a43c38819badb55a21061967ce64
+        uses: docker://quay.io/cilium/cilium-checkpatch@sha256:7047dbb404b1866d841b232793e6b0d54019fb264cef4e6e40884220125c4af4
```

Comment:
I've rebuild all images to force all images to be available in quay.io. I'll give you the new tag once the images are build.

Reply:
@qmonnet `quay.io/cilium/cilium-checkpatch:cc7e6b5811f46d7b040dedfe2f6b0010c2c51a12@sha256:687bd6c550449d2eaa0b258c19703a91093243b9138f8c551563a34df31179f0`

Reply:
Sorry, can you help me with the syntax? I tried
```yaml
        uses: quay.io/cilium/cilium-checkpatch:cc7e6b5811f46d7b040dedfe2f6b0010c2c51a12@sha256:687bd6c550449d2eaa0b258c19703a91093243b9138f8c551563a34df31179f0
```
and
```yaml
        uses: docker://quay.io/cilium/cilium-checkpatch:cc7e6b5811f46d7b040dedfe2f6b0010c2c51a12@sha256:687bd6c550449d2eaa0b258c19703a91093243b9138f8c551563a34df31179f0
```
but none of those works. Should I remove the tag before the `@`?